lib/src/value/mixin.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 Google LLC. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {hash} from 'immutable';
+
+import {Value} from './index';
+
+/** A first-class SassScript mixin. */
+export class SassMixin extends Value {
+  /**
+   * If this mixin is defined in the compiler, this is the unique ID that the
+   * compiler uses to determine which mixin it refers to.
+   *
+   * This is marked as public so that the protofier can access it, but it's not
+   * part of the package's public API and should not be accessed by user code.
+   * It may be renamed or removed without warning in the future.
+   */
+  readonly id: number | undefined;
```

Comment:
I guess this code is copied from `SassFunction`? In `SassFunction`, `id` can be `undefined` because it support both a host function (with signature+callback) and a compiler function (with id). For `SassMixin`, we don't support  host mixin. Therefore, `id` should never be `undefined`.